recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: conda-forge-build-setup
-  version: 4.2.0
+  version: 4.2.1
```

Comment:
We should probably bump the minor version instead as we are adding an environment variable.